lib/livebook/session/data.ex

```diff
@@ -61,18 +61,21 @@ defmodule Livebook.Session.Data do
   @type cell_info :: markdown_cell_info() | code_cell_info() | smart_cell_info()
 
   @type markdown_cell_info :: %{
-          sources: %{primary: cell_source_info()}
+          sources: %{primary: cell_source_info()},
+          comments: list(String.t())
         }
 
   @type code_cell_info :: %{
           sources: %{primary: cell_source_info()},
-          eval: cell_eval_info()
+          eval: cell_eval_info(),
+          comments: list(String.t())
         }
 
   @type smart_cell_info :: %{
           sources: %{primary: cell_source_info(), secondary: cell_source_info()},
           eval: cell_eval_info(),
-          status: smart_cell_status()
+          status: smart_cell_status(),
+          comments: list(String.t())
```

Comment:
We keep the comments in the cell structs, so there's no need for these, right?

Reply:
yep, sorry those are leftovers from my initial exploration that I forgot to clean ;)
fixed in e567cfb